### ISSUE TITLE
Migrating to InvoiceNinja

### ISSUE THREAD

**Issue by santoshkt** [simpleinvoices/simpleinvoices#272]
This is not a bug report in SimpleInvoices. I have been using it for almost 5 years and it is a great software. I made so many customizations over the time for IDs, emails, pdfs to meet my requirements. However, I see the development is stagnant and I am considering to move on to InvoiceNinja self-hosted option. I'm wondering what are the data migration efforts required if anyone has done this before. Will appreciate any inputs on how it is done.

**Comment by apmuthu** [comment]
You're right. The master branch has not had significant changes for quite a while as SI is quite mature for this version. Mobile bootstrap based branches of some forks are being actively developed and so are the ones for PHP 5.6 / 7.x versions.

[InvoiceNinja](https://github.com/invoiceninja/invoiceninja) is based on the Laravel framework with a lot of json/xml/js thrown in along with namespaces. Their core files have not been updated in quite a while too. Compare the sql diffs in SI with [those of InvoiceNinja](https://github.com/invoiceninja/invoiceninja/blob/master/database/setup.sql) and write your migrate scripts studying them

**Comment by santoshkt** [comment]
Thank you for the inputs. Is there a reason why simpleinvoices 2013 version in still beta. I mean its already 4 years.. Do you have any plans on making a general release anytime soon ?

**Comment by apmuthu** [comment]
There have been [356 commits](https://github.com/simpleinvoices/simpleinvoices/compare/2013.1.beta.4...master) since the release of [2013.1 beta 4](https://github.com/simpleinvoices/simpleinvoices/releases/tag/2013.1.beta.4) in 2013-09-30 by @justinkelly .

Releases are made by the project owner, @justinkelly .

You can use the [bleeding edge](https://github.com/simpleinvoices/simpleinvoices/archive/master.zip) however, without waiting for a release.
